modules/serialize/test/ecdsa_signatures.test.ts

```diff
@@ -347,1240 +165,206 @@ function test() {
+  224, 243, 133, 233, 162, 55, 22, 203, 124, 69, 231, 1, 190, 191, 175, 158,
+  82, 80, 168, 172, 29, 97, 13, 141, 126, 184, 238, 159, 214, 213, 92, 114,
+  94, 61, 82,
 ])
 
 // length == 104 both r and s are padded
+// prettier-ignore
 const derSigBothSandRPadded = new Uint8Array([
-  48,
-  102,
-  2,
-  49,
-  0,
-  161,
-  31,
-  228,
-  163,
-  249,
-  149,
-  236,
-  238,
-  15,
-  140,
-  163,
-  28,
-  152,
-  199,
-  168,
-  83,
-  187,
-  60,
-  79,
-  26,
-  71,
-  243,
-  120,
-  0,
-  44,
-  200,
-  217,
-  82,
-  162,
-  181,
-  168,
-  194,
-  181,
-  56,
-  20,
-  193,
-  213,
-  40,
-  112,
-  59,
-  13,
-  254,
-  55,
-  177,
-  231,
-  189,
-  128,
-  71,
-  2,
-  49,
-  0,
-  241,
-  232,
-  224,
-  60,
-  113,
-  203,
-  248,
-  143,
-  34,
-  63,
-  98,
-  221,
-  156,
-  143,
-  58,
-  106,
-  169,
-  169,
-  63,
-  126,
-  103,
-  145,
-  63,
-  246,
-  255,
-  32,
-  74,
-  11,
-  197,
-  255,
-  13,
-  244,
-  105,
-  188,
-  157,
-  210,
-  200,
-  36,
-  140,
-  218,
-  1,
-  115,
-  99,
-  255,
-  212,
-  71,
-  156,
-  38,
+  48, 102, 2, 49, 0, 161, 31, 228, 163, 249, 149, 236, 238, 15, 140, 163,
+  28, 152, 199, 168, 83, 187, 60, 79, 26, 71, 243, 120, 0, 44, 200,
+  217, 82, 162, 181, 168, 194, 181, 56, 20, 193, 213, 40, 112, 59, 13,
+  254, 55, 177, 231, 189, 128, 71, 2, 49, 0, 241, 232, 224, 60, 113,
+  203, 248, 143, 34, 63, 98, 221, 156, 143, 58, 106, 169, 169, 63, 126, 103,
+  145, 63, 246, 255, 32, 74, 11, 197, 255, 13, 244, 105, 188, 157, 210, 200,
+  36, 140, 218, 1, 115, 99, 255, 212, 71, 156, 38,
 ])
+// prettier-ignore
 const rawSigBothSandRPadded = new Uint8Array([
-  161,
-  31,
-  228,
-  163,
-  249,
-  149,
-  236,
-  238,
-  15,
-  140,
-  163,
-  28,
-  152,
-  199,
-  168,
-  83,
-  187,
-  60,
-  79,
-  26,
-  71,
-  243,
-  120,
-  0,
-  44,
-  200,
-  217,
-  82,
-  162,
-  181,
-  168,
-  194,
-  181,
-  56,
-  20,
-  193,
-  213,
-  40,
-  112,
-  59,
-  13,
-  254,
-  55,
-  177,
-  231,
-  189,
-  128,
-  71,
-  241,
-  232,
-  224,
-  60,
-  113,
-  203,
-  248,
-  143,
-  34,
-  63,
-  98,
-  221,
-  156,
-  143,
-  58,
-  106,
-  169,
-  169,
-  63,
-  126,
-  103,
-  145,
-  63,
-  246,
-  255,
-  32,
-  74,
-  11,
-  197,
-  255,
-  13,
-  244,
-  105,
-  188,
-  157,
-  210,
-  200,
-  36,
-  140,
-  218,
-  1,
-  115,
-  99,
-  255,
-  212,
-  71,
-  156,
-  38,
+  161, 31, 228, 163, 249, 149, 236, 238, 15, 140, 163, 28, 152, 199, 168, 83,
+  187, 60, 79, 26, 71, 243, 120, 0, 44, 200, 217, 82, 162, 181, 168,
+  194, 181, 56, 20, 193, 213, 40, 112, 59, 13, 254, 55, 177, 231, 189,
+  128, 71, 241, 232, 224, 60, 113, 203, 248, 143, 34, 63, 98, 221, 156,
+  143, 58, 106, 169, 169, 63, 126, 103, 145, 63, 246, 255, 32, 74, 11, 197,
+  255, 13, 244, 105, 188, 157, 210, 200, 36, 140, 218, 1, 115, 99, 255, 212,
+  71, 156, 38,
+])
+
+// Suite AlgorithmSuiteIdentifier.ALG_AES256_GCM_IV12_TAG16_HKDF_SHA512_COMMIT_KEY_ECDSA_P384
+// prettier-ignore
+const derSigOnlySPadded = new Uint8Array([
```

Comment:
**Only** concerns me, as it raises confusion. Given all I know, I assume that *derSigOnlySPadded* is different from *derSigSPadded*, and the difference is that `r = 0`, is that right? If not, what is the difference?

Reply:
I don't like to delete vectors.  How about numbers? `derSigSPadded` `derSigSPadded1` derSigSPadded2`?